tests/container_hardening/test_container_hardening.py

```diff
@@ -10,16 +10,20 @@
 
 NO_PRIVILEGED_CONTAINERS = [
     'bgp',
+    'lldp',
+    'teamd'
 ]
 
 
-def test_container_privileged(duthost):
+def test_container_privileged(duthost, enum_rand_one_asic_index):
```

Comment:
another comment which I missed earlier, duthost will select only one duthost.
For multi-dut scenario, we can do:
def test_container_privileged(duthosts, enum_rand_one_per_hwsku_hostname, enum_rand_one_asic_index):
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
     
Rest can remain as is.
This way test will run for each hwsku in testbed and in each selected dut, the test will be executed on one random asic_index.
Could refer bgp/test_bgp_fact.py.

Reply:
Thanks for such a clear explanation.